[2022-02-19 06:24:43,259] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: demo_etl.Load_Data.Save_to_DB manual__2022-02-19T06:24:16.178511+00:00 [queued]>
[2022-02-19 06:24:43,269] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: demo_etl.Load_Data.Save_to_DB manual__2022-02-19T06:24:16.178511+00:00 [queued]>
[2022-02-19 06:24:43,269] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-19 06:24:43,269] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-19 06:24:43,269] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-19 06:24:43,282] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): Load_Data.Save_to_DB> on 2022-02-19 06:24:16.178511+00:00
[2022-02-19 06:24:43,288] {standard_task_runner.py:52} INFO - Started process 31222 to run task
[2022-02-19 06:24:43,351] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'demo_etl', 'Load_Data.Save_to_DB', 'manual__2022-02-19T06:24:16.178511+00:00', '--job-id', '263', '--raw', '--subdir', 'DAGS_FOLDER/demo_etl.py', '--cfg-path', '/tmp/tmp05zpr969', '--error-file', '/tmp/tmp4c01l11o']
[2022-02-19 06:24:43,352] {standard_task_runner.py:77} INFO - Job 263: Subtask Load_Data.Save_to_DB
[2022-02-19 06:24:43,611] {logging_mixin.py:109} INFO - Running <TaskInstance: demo_etl.Load_Data.Save_to_DB manual__2022-02-19T06:24:16.178511+00:00 [running]> on host c39e331ab017
[2022-02-19 06:24:43,894] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=demo_etl
AIRFLOW_CTX_TASK_ID=Load_Data.Save_to_DB
AIRFLOW_CTX_EXECUTION_DATE=2022-02-19T06:24:16.178511+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-19T06:24:16.178511+00:00
[2022-02-19 06:24:43,905] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-02-19 06:24:43,906] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'next_ds' from the template is deprecated and will be removed in a future version. Please use '{{ data_interval_end | ds }}' instead.
[2022-02-19 06:24:43,913] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'next_ds_nodash' from the template is deprecated and will be removed in a future version. Please use '{{ data_interval_end | ds_nodash }}' instead.
[2022-02-19 06:24:43,914] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'next_execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_end' instead.
[2022-02-19 06:24:43,915] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'prev_ds' from the template is deprecated and will be removed in a future version.
[2022-02-19 06:24:43,916] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'prev_ds_nodash' from the template is deprecated and will be removed in a future version.
[2022-02-19 06:24:43,917] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'prev_execution_date' from the template is deprecated and will be removed in a future version.
[2022-02-19 06:24:43,918] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'prev_execution_date_success' from the template is deprecated and will be removed in a future version. Please use 'prev_data_interval_start_success' instead.
[2022-02-19 06:24:43,919] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'tomorrow_ds' from the template is deprecated and will be removed in a future version.
[2022-02-19 06:24:43,920] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'tomorrow_ds_nodash' from the template is deprecated and will be removed in a future version.
[2022-02-19 06:24:43,921] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'yesterday_ds' from the template is deprecated and will be removed in a future version.
[2022-02-19 06:24:43,922] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'yesterday_ds_nodash' from the template is deprecated and will be removed in a future version.
[2022-02-19 06:24:43,986] {logging_mixin.py:109} INFO - Done !!
[2022-02-19 06:24:44,259] {logging_mixin.py:109} INFO - Done !!
[2022-02-19 06:24:44,261] {logging_mixin.py:109} INFO - Empty DataFrame
Columns: [VendorID, lpep_pickup_datetime, lpep_dropoff_datetime, RatecodeID, PULocationID, DOLocationID, passenger_count, trip_distance, fare_amount, extra, mta_tax, tip_amount, tolls_amount, improvement_surcharge, total_amount, payment_type, trip_type, congestion_surcharge]
Index: []
[2022-02-19 06:24:46,153] {logging_mixin.py:109} INFO - Save Data to Database Done !!
[2022-02-19 06:24:46,153] {python.py:175} INFO - Done. Returned value was: None
[2022-02-19 06:24:46,165] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=demo_etl, task_id=Load_Data.Save_to_DB, execution_date=20220219T062416, start_date=20220219T062443, end_date=20220219T062446
[2022-02-19 06:24:46,218] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-19 06:24:46,258] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
